src/Package/Impl/Plots/PlotContentProvider.cs

```diff
@@ -31,7 +31,9 @@ internal sealed class PlotContentProvider : IPlotContentProvider {
         private async void RSession_Connected(object sender, EventArgs e) {
             // Let the host know the size of plot window
             if (_lastPixelWidth >= 0 && _lastPixelHeight >= 0) {
-                await ApplyNewSize();
+                try {
+                    await ApplyNewSize();
+                } catch (OperationCanceledException) { } catch (MessageTransportException) { } catch (RException) { }
```

Comment:
I don't think we're still doing it, actually, if you're doing interactions - i.e. stderr output should no longer translate to `RException` from `RespondAsync`. @AlexanderSher, is that right?